### DISCUSSION TITLE
unable to run amplication on local env.

### DISCUSSION THREAD

**Discussion by Suksham-sharma** [amplication/amplication#6199] [Q&A] [answered]
i followed the steps , and i'm not sure what is to happen after the docker command , i'm working on github octernships, and this is my first time using amplication , and i have been assigned to create a new plugin ,  i read all the docs , it's still unclear , will anyone be willing to help and just guide me lil bit . I'll be very thankful .

**Comment by overbit** [comment, marked as answer]
Feel free to raise and issue in case this still happen
